app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    @posts = @user.posts.includes([:comments])
     @comments = Comment.all
   end
 
```

Comment:
- [ ] Please kindly use `CanCanCan` for Post authorization in Post Controller. You can achieve that by including the below code before the methods

``load_and_authorize_resource``

Reply:
Hello @AnselemOdims we tried to add that before methods and it raised an error